ccf_transaction_processor/test/ping_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Intel Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+from infra.clients import CCFClient
+import argparse
+import time
+import os
+
+
+def run(args):
+    # SNIPPET_START: parsing
+
+    num_pings = args.num_pings
+    host = args.host
+    port = 6006
```

Comment:
given that i have to specify the port for creating the TP, it would have seen more natural that the argument to --host is also of the form <ip:port>. In fact, calling it `--node` in that case would be even easier ....